### PR TITLE
Proposal to fix #4831 (CZ TTS problem with PREC1 and units)

### LINKED ISSUE #4831: Czech speech audio issue 2.2N369
There is issue in 2.2N 364-369. Selected czech sound pack not reproduce the values well. 
the value 10.7V should say "ten point seven volts" 
we got only  "ten seven"

English audio works well. Tested on X7, X9E 

On SD we have sthe sound files well and inclide point, volts ....

### PR DESCRIPTION
**Pull request by bsongis**

### CODE REVIEW

**Comment by mhotar** [comment]
10 volts is OK:
playFile("/SOUNDS/cz/SYSTEM/0010.wav", flags=0, id=1)
playFile("/SOUNDS/cz/SYSTEM/volt2.wav", flags=0, id=1)

but 10.1volts is not:
playFile("/SOUNDS/cz/SYSTEM/0010.wav", flags=0, id=1)
pushUnit: out of bounds unit : 114
playFile("/SOUNDS/cz/SYSTEM/0001.wav", flags=0, id=1)
playFile("/SOUNDS/cz/SYSTEM/mamp0.wav", flags=0, id=1)

**Comment by bsongis** [comment]
mamp0.wav, then we need another -1 to play volts instead of amps, right?

**Comment by 3djc** [comment]
Seems ok to me

**Comment by bsongis** [comment]
Just this big switch ... we should replace the values by constants!

File: radio/src/translations/tts_cz.cpp [hunk 1]
```diff
@@ -42,9 +42,9 @@ enum CzechPrompts {
 #if defined(VOICE)
 
 #if defined(CPUARM)
-  #define CZ_PUSH_UNIT_PROMPT(p, u) cz_pushUnitPrompt((u), (p), id)
+  #define CZ_PUSH_UNIT_PROMPT(p, u) cz_pushUnitPrompt((p), (u), id)
```
Comment (bsongis) [inline radio/src/translations/tts_cz.cpp]: it should be u first, no? (on both sides!)

**Comment by bsongis** [comment]
:+1 Thanks!

**Pull request merged**